templates/frontend/objects/article_details.tpl

```diff
@@ -212,24 +212,26 @@
 							{translate key="submission.authorBiography"}
 						{/if}
 					</h2>
+					<ul class="authors">
 					{foreach from=$publication->getData('authors') item=author}
 						{if $author->getLocalizedData('biography')}
-							<section class="sub_item">
-								<h3 class="label">
+							<li class="sub_item">
+								<span class="label">
```

Comment:
Are you sure we don't want a `<h3>` here? I think even within a list it is good to keep heading hierarchy where it makes sense.

Reply:
I was applying the same approach of listing authors we have on the top. We don't use headings on the list itself. Should we consider the author's name a heading section of the document? I can see it could be interchangeable with `dl > dt/dd` elements which also conflicts with the Heading approach. 
See the top:
![image](https://user-images.githubusercontent.com/4992902/212893080-5eff58d2-471d-41e9-9504-3c0039a8b086.png)

And the bottom:
![image](https://user-images.githubusercontent.com/4992902/212893187-a1e02f3d-3921-4d5b-9e9c-6bd14db6256f.png)

Reply:
I was reviewing our other issues related to this topic, and this one is related to the heading removal from listing author biographies: https://github.com/pkp/pkp-lib/issues/6901

Reply:
Ok, if the auditor suggested a list with no headings then let's go with that.

Reply:
Perfect, I will use `DIV` as suggested by the auditor to keep it in block elements.